content/applications/hr/recruitment/new_job.rst

```diff
@@ -143,68 +173,274 @@ Sections are indicated by a gray background, while questions have a white backgr
+
+.. image:: new_job/single-line.png
+   :align: center
+   :alt: The three possible checkboxes that can appear if a single line of tex tis selected for the
+         question type.
+
+Matrix
+^^^^^^
+
+For a :guilabel:`Matrix` :guilabel:`Question Type`, there are two sets of data that need to be
+input. The rows and columns must both be configured. The columns are represented by the
+:guilabel:`Choices` section, while the rows are configured in the :guilabel:`Rows` section.
+
+The method for populating both sections is the same. In the :guilabel:`Answers` tab, click
+:guilabel:`Add a line` in either the :guilabel:`Choices` or :guilabel:`Row` section.  A line
+appears, and an answer can be entered. After typing in the answer, click off the line, or press
+enter to lock in the new answer on the form and have another answer line appear. Repeat this for all
+answers for both the :guilabel:`Choices` and :guilabel:`Rows` sections.
+
+.. figure:: new_job/matrix.png
+   :align: center
+
+   This is an example matrix that asks an applicant what shifts they are available to work on
+   Saturdays and Sundays, either morning, afternoon, or evening.
+
+Description
+***********
+
+Enter any information that may be helpful to an applicant. This description will appear for this
+specific question only, and therefore should be question-specific and not generalized.
+
+Options
+*******
+
+To view the options that are available to set for a question, click on the :guilabel:`Options` tab.
+The :guilabel:`Layout`, :guilabel:`Constraints`, and :guilabel:`Live Sessions` sections are
+universal for all :guilabel:`Question Types`, while the :guilabel:`Answers` tab is specific to the
+:guilabel:`Question Type` selected, and changes based on the selection.
+
+Answers
+^^^^^^^
+
+- :guilabel:`Multiple choice: only one answer` and :guilabel:`Multiple choice: multiple answers
+  allowed`: A :guilabel:`Show Comments Field` option appears. Activate the option to allow the
+  applicant to answer the question by typing in an answer (or comment). A :guilabel:`Comment
+  Message` and :guilabel:`Comment is an answer` field appear if activated. Enter the text to appear
+  in the :guilabel:`Comment Message` field. This should be helpful to the applicant, such as `If
+  other, please specify`. Last, if the comment should be logged as the applicant's answer, activate
+  the box.
+- :guilabel:`Multiple Lines Text Box`: A :guilabel:`Placeholder` field appears. Enter any text that
+  should appear with the question to help clarify how the applicant should answer.
+- :guilabel:`Single Line Text Box` and :guilabel:`Numerical Value`: A :guilabel:`Validate entry`
+  option appears. Activate this option if what the applicant enters must be a numerical value, *and*
+  needs to be verified. If the answer is *not* a numerical value, the answer cannot be verified (do
+  not activate the :guilabel:`Validate entry` option). When activated, several other options appear.
```

Comment:
Double negative... I would rephrase as "can **only** be verified if answer is numerical" or something similar